omegatrade-app/backend/app/models/user.model.js

```diff
@@ -0,0 +1,51 @@
+'user strict';
+const database = require('./../config/database.js');
+var User = function (user) {};
+
+User.registerUser = async function (user, cb) {
+  database.runTransaction(async (err, transaction) => {
+    if (err) {
+      cb(err, null)
+      return;
+    }
+    try {
+      const [rowCount] = await transaction.runUpdate({
```

Comment:
Do you need to use DML here? The `rowCount` will always be 1, as the statement will either succeed and insert 1 row, or fail and return an error.